### PR TITLE
secuTrialR recipe

### PR DESCRIPTION
**Pull request by PatrickRWright**
We created the R package secuTrialR for interactinging with clinical trials databases and I would like to add it to Bioconda because some people may like to use it in this environment at some point.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/r-secutrialr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  url: "{{ github }}/archive/{{ version }}.tar.gz"
+  sha256: 9dae54b415cf94ed1d3a386ea60415dacaaa7737a17e37e9aa424e668ddaab46
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  host:
+    - r-base >=3.5
```
Comment (bgruening) [inline recipes/r-secutrialr/meta.yaml:18]: This pinning can be removed.

File: recipes/r-secutrialr/build.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+$R CMD INSTALL --build .
```
Comment (bgruening) [inline recipes/r-secutrialr/build.sh:1]: this simple one could be moved inside the meta.yml as a script tag into the build section.

Comment (PatrickRWright) [reply recipes/r-secutrialr/build.sh:1]: like this?

```
build:
  script: $R CMD INSTALL --build .
  noarch: generic
  number: 0

```

**Comment by PatrickRWright** [comment]
Hi @bgruening if you want I can look into the conda forge suggestion. This is, however, a purely life-sciences related package and thus seems to fit.

**Comment by PatrickRWright** [comment]
Any idea what may be causing the problem? I don't understand what `BIOCONDA ERROR` is telling me. I simply copied someone else's recipe for an R package on github and adapted accordingly.

**Pull request closed without merging**